### PR TITLE
PFM-987 :: Works UI Deployment configured

### PR DESCRIPTION
**Pull request by jagankumar-egov**

### CODE REVIEW

File: deploy-as-code/helm/charts/digit-works/frontend/works-ui/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
+
+dependencies:
+- name: common
+  version: 0.0.5
+  repository: file://../../../common
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/digit-works/frontend/works-ui/Chart.yaml:26]: I think the common directory is present one level above this. 
We will need an extra `../`. 
Please verify and make the change.

File: deploy-as-code/helm/charts/digit-works/frontend/works-ui/values.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+image:
+  repository: "works-ui"
+replicas: "1"
+httpPort: 80
+healthChecks:
+  enabled: true
+  livenessProbePath: "/works-ui/"
+  readinessProbePath: "/works-ui/"
+
+namespace: egov
```
Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/digit-works/frontend/works-ui/values.yaml:24]: Move the namespace attribute to the first few lines of the file.

Comment (rushang7-eGov) [inline deploy-as-code/helm/charts/digit-works/frontend/works-ui/values.yaml:24]: Move this to the `works` namespace.

File: deploy-as-code/helm/charts/digit-works/frontend/works-ui/Chart.yaml (hunk 1 above)
Comment (jagankumar-egov) [reply deploy-as-code/helm/charts/digit-works/frontend/works-ui/Chart.yaml:26]: Yeah i have added extra

**Review by rushang7-eGov — APPROVED** [review]

**Pull request merged**